tocrify/data/mets2hocr.yml

```diff
@@ -1,13 +1,21 @@
 chapter:
     0: "ocr_chapter"
     1: "ocr_section"
-    2: "ocr_subsection"
-    3: "ocr_subsubsection"
+    2: "ocr_subsect1"
```

Comment:
I think it's good that you follow the ISO 12083 Book XML DTD when structuring the chapters etc. see: http://xml.coverpages.org/iso12083xmlbookdtd19990125.html